targets/TARGET_Samsung/TARGET_SIDK_S5JS100/BINARIES/bootarg_Board_0.2.bin

```diff
@@ -0,0 +1 @@
+BOARD=0.2

```

Comment:
Is this a binary file?

Reply:
Addressed from https://github.com/ARMmbed/mbed-os/pull/10170/files#diff-6ab48777b97d4bd41471af54d2b19faeR1